lighthouse/classes/biosero.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+from lighthouse.classes.automation_system import AutomationSystem
+from lighthouse.classes.events.biosero import (
```

Comment:
should there be an import for destination partial?

Reply:
Yes thanks, good spot, I've added it.